Firmware/MotorControl/motor.cpp

```diff
@@ -192,6 +192,21 @@ float Motor::effective_current_lim() {
     return current_lim;
 }
 
+float Motor::max_available_torque() {
+    //return the maximum available torque for the motor.
+    //Note - for ACIM motors, available torque is allowed to be 0.
```

Comment:
By convention we usually put general function documentation right above the function definition rather than right below.